index.html

```diff
@@ -13,39 +13,39 @@
 </head>
 <body>
     <header>
-        <section class="header">
-            <img src="/assets/img/logos/logo icon/who-travels-icon.svg" alt="Who Travels Logo" class="logo">
-            <nav>
-                <div class="mobile-menu">
-                    <input type="checkbox">
-                    <span class="hamburger"></span>
-                    <span class="hamburger"></span>
-                    <span class="hamburger"></span>
+        <section class="hero">
+            <div class="hero-img-box">
+                <img class="hero-img" src="/assets/img/heros/hero-home.jpg" alt="photo of man looking at stars">
+            </div>
+            <div class="hero-content">
+                <span class="hero-text">
+                    <h1 class="hero-h1">Travel</h1>
+                    <h2 class="hero-h2"> Who Travels?</h2>
+                    <p class="hero-body">It could be you! We're the one stop shop for all time and space travel needs.</p>
+                </span>
+            </div>
+        </section>
 
+        <section>
+            <div class="nav-container">
+                <img src="/assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo">
+                <nav class="nav">
```

Comment:
we will have to eventually rename this class, since classes shouldn't be the names of html attributes (ex: `nav`, `header`, `h1`, etc) it's fine for now though, we can do that in a cleanup once everything is up.

Reply:
I changed it to 'nav-menu' and it should reflect in the CSS as well